src/styles/index.scss

```diff
@@ -1,3 +1,97 @@
-body {
+@import 'mixin';
+@import 'stars';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto,sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding: 32px 16px 16px;
+
+    &__image {
+      background-image: url('./../images/imac.jpeg');
+      background-size: cover; 
+      width: 160px;
+      height: 134px;
+      margin: 0 auto 40px;  
+    }
+    &__name {
+      display: block;
+      color: $main-accent;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      text-align: center;
+      margin-bottom: 4px;
+    }
+    &__code {
+      display: block;
+      color: $secondary;
+      margin-bottom: 16px;
+    }
+    &__reviews {
+      height: 16px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center; 
+      margin-bottom: 24px;
+    }
+    &__price {
+      display: flex;
+      justify-content: space-between;
+      margin-bottom: 16px;
+    }
+    &__button {
+      color: $white;
+      background-color: $blue-accent;
+      border: none;
+      border-radius: 5px;
+      width: 100%;
+      height: 40px;
+      text-align: center;
+      font-weight: 700;
+      line-height: 16px;
+      font-size: 14px;
+
+      &:hover {
+        background: #FFF;
+        color: $blue-accent;
+        border: 1px solid $blue-accent;
+      }
+    }
+}
+
+.reviews {
+  &__stars {
+    display: flex;
+  }
+  &__counter {
+    color: $main-accent;
+  }
+}
+
+.price {
+  &__text {
+    font-size: 12px;
+    line-height: 18px;
+    color:  $secondary;
+  }
+  &__numbers {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent;
+  }
+}
```

Comment:
It should be a part of card block you must have only two blocks `card` and `stars`